src/vmm/src/builder.rs

```diff
@@ -608,6 +609,14 @@ fn attach_legacy_devices_aarch64(
         .map_err(Error::RegisterMMIODevice)
 }
 
+#[cfg(target_arch = "aarch64")]
+fn init_vcpus(vm: &Vm, vcpus: &[Vcpu]) -> super::Result<()> {
+    for vcpu in vcpus {
+        vcpu.kvm_vcpu.init(vm.fd()).map_err(Error::VcpuInit)?;
```

Comment:
Would it make sense to bake this `init()` in `aarch64::Vcpu::new`?

Reply:
Indeed, I baked it into `Vcpu::new`, take a look.